### ISSUE TITLE
CSS et filtrage

### ISSUE THREAD

**Issue by abelards** [simonc/jeveuxapprendreruby.fr#8]
Je propose de :
- mettre la couleur principale pour chaque DD et une classe selon le contenu du DL
- peut-être les mettre sur 2 lignes (3 colonnes) pour gagner de la place
- mettre un peu de JS pour faire un filtrage selon les valeurs des tags

Exemple : http://maitre-du-monde.fr/koans/bestof.html
 lignes 70 à 130 https://github.com/abelards/abelards.github.com/blob/master/koans/bestof.html#L70

**Comment by simonc** [comment]
@abelards tu parles de tags ou je suis à côté de la plaque ?

**Comment by simonc** [comment]
On pourrait avoir un truc qui ressemble plus ou moins à ça :

![slice 1](https://cloud.githubusercontent.com/assets/119303/12217697/b641656a-b708-11e5-96bf-ad8b3a9920b8.png)

**Comment by abelards** [comment]
Complètement ! Ce serait pas énorme ?

**Comment by abelards** [comment]
Je propose aussi :
- `level` passer de "tous" à la combo des trois [débutant, intermédiaire, avancé]  
- `categories` passer de "puzzles" à "algo"
- `url` passe sous le titre au lieu d'avoir un "tag"
  ![screen shot 2016-01-10 at 16 21 40](https://cloud.githubusercontent.com/assets/584600/12222225/64761eac-b7b6-11e5-9bca-fb5cc6df129f.png)

**Comment by simonc** [comment]
Je suis d'accord avec l'url en dessous du titre et pour passer à algo, d'ailleurs je vire "puzzles" de _sujets_ ça n'y a pas tellement sa place.

J'aime moyen les labels en couleur pour être honnête. et en ce qui concerne le dégradé de couleur c'est un essai où c'est ce que tu voudrais obtenir ? J'ai peur qu'avec les boucles jekyll etc ça fasse faire pas mal de choses en dur, je sais pas ce que tu en penses.

D'ailleurs j'hésite à virer jekyll parce que sur gh-pages pas de plugin autorisé et liquid me sort par les yeux quand je veux avoir la main (genre pas de flat_map pour récupérer la liste des sujets/modalités/...)

**Comment by abelards** [comment]
On 10 Jan 2016 4:43 pm, "Simon Courtois" notifications@github.com wrote:

> Je suis d'accord avec l'url en dessous du titre et pour passer à algo,
> d'ailleurs je vire "puzzles" de sujets ça n'y a pas tellement sa place.

OK.

> J'aime moyen les labels en couleur pour être honnête.
> Pas de soucis.
> 
> et en ce qui concerne le dégradé de couleur c'est un essai où c'est ce
> que tu voudrais obtenir ?
> C'est mon but mais si tu n'aimes pas on zappe.
> 
> J'ai peur qu'avec les boucles jekyll etc ça fasse faire pas mal de choses
> en dur, je sais pas ce que tu en penses.

J'en ai fait une fois. Je chercherai pour le refaire en Jekyll seul et sans
plugin pour la compatibilité.

> D'ailleurs j'hésite à virer jekyll parce que sur gh-pages pas de plugin
> autorisé et liquid me sort par les yeux quand je veux avoir la main (genre
> pas de flat_map pour récupérer la liste des sujets/modalités/...)

Voilà voilà. Si on y arrive vraiment pas why not.
